### PR TITLE
Add condition for headings without text or a11y name to error flag cr…

### PR DESCRIPTION
**Pull request by lynnerang**
Teeny update that adds the error state flag to item headers when the role is 'heading' and it doesn't have visible text or an a11y name.

Screen Shot 2021-12-15 at 2 46 03 PM

### CODE REVIEW

File: specter2/src/views/Annotations/Annotations.svelte [hunk 1]
```diff
@@ -44,6 +44,7 @@
     const { labels, role } = item;
     return (
       (role === 'image' && !labels.alt)
+      || (role === 'heading' && !(labels.a11y || labels.visible))
```
Comment (joeyyang) [inline specter2/src/views/Annotations/Annotations.svelte:47]: nice - since this check is getting long, can you break this into separate variables?

**Review by joeyyang — APPROVED** [review]
Nice, readability request but that's it!

**Review by gblakeman — APPROVED** [review]
🎉👊

File: specter2/src/views/Annotations/Annotations.svelte (hunk 1 above)
Comment (lynnerang) [reply specter2/src/views/Annotations/Annotations.svelte]: sho thang, what you think about this (just pushed update)?

Comment (joeyyang) [reply specter2/src/views/Annotations/Annotations.svelte]: love it!

Comment (lynnerang) [reply specter2/src/views/Annotations/Annotations.svelte]: sweeet, thanks!

**Pull request merged**